types/sentry.d.ts

```diff
@@ -0,0 +1,39 @@
+import { Options as WebpackOptions } from 'webpack'
+import { Options as SentryOptions } from '@sentry/types'
+import { SentryCliPluginOptions } from '@sentry/webpack-plugin'
+
+export type IntegrationsConfiguration = Record<string, unknown>
+
+export interface LazyConfiguration {
+    chunkName?: string
+    injectLoadHook?: boolean
+    injectMock?: boolean
+    mockApiMethods?: boolean | string[]
+    webpackPrefetch?: boolean
+    webpackPreload?: boolean
+}
+
+export interface ModuleConfiguration {
+    attachCommits?: boolean
+    clientConfig?: SentryOptions
+    clientIntegrations?: IntegrationsConfiguration
+    config?: SentryOptions
+    disableClientRelease?: boolean
+    disableClientSide?: boolean
+    disabled?: boolean
+    disableServerRelease?: boolean
+    disableServerSide?: boolean
+    dsn?: string | ''
+    initialize?: boolean
+    lazy?: boolean | LazyConfiguration
+    logMockCalls?: boolean
+    publishRelease?: boolean
+    repo?: string | ''
```

Comment:
```suggestion
    repo?: string
```

`''` is of type `string` 

![image](https://user-images.githubusercontent.com/25272043/88660635-676d8c00-d0d7-11ea-88a4-55c2d4322999.png)